proposals/0000-local-do.rst

```diff
@@ -236,38 +261,61 @@ Related work
   * This was the essence of the `OverloadedDo proposal <https://github.com/ghc-proposals/ghc-proposals/pull/78>`_, though type inference was never solved for this
   * A more recent idea is `supermonads <http://www.cs.nott.ac.uk/~psznhn/Publications/jfp2018.pdf>`_, which solves the type inference issue using a plugin
 
-  It requires somewhat less work (“only” a plugin, rather than a change in GHC's compiler, at least it's more modular), and is more automatic, as the correct functions are picked automatically from the type. But there is no way that this will capture all the desired notion: some restrictions need be imposed for the sake of type inference. Note as well that this proposal doesn't preclude an automatic approach when appropriate: simply import your very automatic builder in scope, and all the do-expressions without an explicit builder will use this.
-
-* There is a way to emulate ``-XLocalDo`` in current GHC using ``-XRecordWildcards``: have no ``(>>=)`` and such in scope, and import a builder with ``Builder {..} = builder``. It is used in `linear-base <https://github.com/tweag/linear-base/blob/0d6165fbd8ad84dd1574a36071f00a6137351637/src/System/IO/Resource.hs#L119-L120>`_. This is not a very good solution: it is rather a impenetrable idiom, and, if a single function uses several builders, it yields syntactic contortion (which is why shadowing warnings are deactivated `here <https://github.com/tweag/linear-base/blob/0d6165fbd8ad84dd1574a36071f00a6137351637/src/System/IO/Resource.hs#L1>`_)
```

Comment:
This paragraph could be kept.

Reply:
Indeed, it is adapted here:
https://github.com/tweag/ghc-proposals/pull/6/files#diff-de5d21226ba42e79ef79485d1f14e1a5R266